tools/spelling/spelling_dictionary.txt

```diff
@@ -37,6 +37,7 @@ Millicores
 NID
 NIST
 NORMALISATION
+STREQ
```

Comment:
I assume that once `EXPECT_THROW_WITH_MESSAGE` is used, this will not be needed.